ossindex/ossindex.go

```diff
@@ -74,14 +86,9 @@ func AuditPackages(purls []string) ([]types.Coordinate, error) {
 
 	// Initialize the cache
 	db, err := openDb(dbDir)
+	defer db.Close()
```

Comment:
I think we might want this after error check?? I'm guessing its possible to have an error returned and db to be nil. So this code will panic and not be a great user experience.

Reply:
In: ce572ea , great catch!